src/Sylius/Bundle/ApiBundle/Validator/Constraints/ChangedItemQuantityInCartValidator.php

```diff
@@ -40,6 +40,12 @@ public function validate($value, Constraint $constraint)
         /** @var ChangedItemQuantityInCart $constraint */
         Assert::isInstanceOf($constraint, ChangedItemQuantityInCart::class);
 
+        if ($value->quantity === null) {
+            $this->context->addViolation($constraint->quantityNotProvided);
+
+            return;
+        }
```

Comment:
I feel like most of these (notBlank/notNull) could be achieved via constraint mapping with minimal/no code changes.

Reply:
I first tried to set up `NotBlank` constraint, but it has to be validated here as well, because of this:
https://github.com/Sylius/Sylius/pull/15072/files#diff-6fa0ebf7574948a5a4bbf107b596f0e76c2c5c36d9cd8533c9c1e16fddffc912R86

Reply:
No longer valid